src/expressive.annotations.validate.js

```diff
@@ -524,13 +524,12 @@ var
                             namespacedEvents.push(typeHelper.string.format('{0}.expressive.annotations', event));
                         }
                     });
-                    $(form).find('input, select, textarea').on(namespacedEvents.join(' '), function(event) {
+                    // attach handlers to all inputs that do not have 'ea-bound' class (unless force == true)
+                    $(form).find('input, select, textarea').not(function(idx,el){ return $(el).hasClass('ea-bound') || force;}).on(namespacedEvents.join(' '), function () {
                         var field = $(this).attr('name');
-                        logger.dump(typeHelper.string.format('Dependency validation trigger - {0} event, handled.', event.type));

```

Comment:
I think we can leave this line here untouched, because it just gives some view of what's currently going on if debug is enabled.